lib/Doctrine/Common/DataFixtures/Executor/AbstractExecutor.php

```diff
@@ -139,3 +140,5 @@ public function purge()
      */
     abstract public function execute(array $fixtures, $append = false);
 }
+
+interface_exists(ObjectManager::class);
```

Comment:
Why is this line included? It appears a couple of other places too, and I don't see why it is necessary.

Reply:
It really isn't. If you want to know, follow the link in the PR body / the commit message.